README.rdoc

```diff
@@ -274,29 +262,43 @@ This is what we do behind the scenes when you do +deliver+
 
   If not, show a list of authors that need to provide a +LGTM+.
   
-If the review is done, it's time to merge. Here's what happens:
+* If the review is done, it's time to merge. Here's what happens:
 
-First, update our local +master+ so we don't get conflicts
-  git checkout master
-  git pull origin master
+  First, we use the github_api gem to merge the pull request. 
 
-Next, squash merge our feature branch
-  git merge --squash nh-branch-name
-  
-Now, we'll apply a little magic so we have a great commit message by default. Your editor will open and you'll see a nice default for your commit message based on the pull request body.
+  We call the public API for: 
+
+    github.pull_requests.merge 'user-name', 'repo-name', 'number', payload
 
-Once you've saved the commit, prompt the user to see if we should continue
-  Merge complete!
-  Would you like to push this branch to your remote repo and cleanup your feature branch?
+    Please take a look at lib/git_reflow/git_server/git_hub/pull_request.rb:102-107 for an example of the call.
+    This call makes an HTTP request using the Github API to merge the available pull request passing in the user name, repository name, pull request number, and some additional data in the payload.
+
+    The payload contains data in the following format:
+
+    data = {
+      "commit_title",
+      "commit_message",
```

Comment:
Absolutely.  It is a big reason why we decided to use git reflow.  We want to have a custom deliver template that the user will be able to edit.  We want to retain this functionality in all scenarios.